plugins/ufm_events_grafana_dashboard_plugin/scripts/init.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#
+# Copyright © 2013-2024 NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+#
+# This software product is a proprietary product of Nvidia Corporation and its affiliates
+# (the "Company") and all right, title, and interest in and to the software
+# product, including all associated intellectual property rights, are and
+# shall remain exclusively with the Company.
+#
+# This software product is governed by the End User License Agreement
+# provided with the software product.
+# @author: Anan Al-Aghbar
+# @date:   July 18, 2024
+#
+set -eE
+PLUGIN_NAME=ufm_events_grafana_dashboard
+SRC_DIR_PATH=/opt/ufm/ufm_plugin_${PLUGIN_NAME}/${PLUGIN_NAME}_plugin
+CONFIG_PATH=/config
+
+cp -r ${SRC_DIR_PATH}/conf/* ${CONFIG_PATH}
+
+# UFM version test
+required_ufm_version=(6 12 0)
```

Comment:
Maybe get it as an env param, from outside. So this config will be in another central location.

Reply:
fixed